runtime/src/main/cpp/ObjCExport.mm

```diff
@@ -438,9 +438,9 @@ static OBJ_GETTER(boxedBooleanToKotlinImp, NSNumber* self, SEL cmd) {
       reinterpret_cast<struct Block_descriptor_1_without_helpers*>(literal->descriptor)->signature;
 }
 
-// Note: replaced by compiler in appropriate compilation modes.
-__attribute__((weak)) convertReferenceFromObjC* Kotlin_ObjCExport_blockToFunctionConverters = nullptr;
-__attribute__((weak)) int Kotlin_ObjCExport_blockToFunctionConverters_size = 0;
+// Note: defined by compiler.
+extern "C" convertReferenceFromObjC* Kotlin_ObjCExport_blockToFunctionConverters;
```

Comment:
Maybe put all `extern "C"` in single clause?

Reply:
Unfortunately
```c++
extern "C" {
    int x;
}
```
is not the same as
```c++
extern "C" int x;
```
in C++.